### ISSUE TITLE
possible bug in 'abyss-bloom kmers -r'?

### ISSUE THREAD

**Issue by notestaff** [bcgsc/abyss#156]
@benvvalk 
At https://github.com/bcgsc/abyss/blob/master/Bloom/bloom.cc#L787
```c++
if (bloom[Kmer(kmer)] || opt::inverse) {
```
should it be
```c++
if ((bloom[Kmer(kmer)] != opt::inverse) {
```
?

**Comment by benvvalk** [comment]
@notestaff 

Yes, you are right! I've pushed the fix to our private dev repo, so that it will be included in the next release.

Thanks!

**Comment by benvvalk** [comment]
fixed by abyss-dev/ca8f5246
